examples/wifi/scan/README.md

```diff
@@ -1,11 +1,11 @@
 # Wifi SCAN Example
 
-This example shows how to use scan of ESP32.
+This example shows how to use the scan functionality of ESP32.
 
-We have two way to scan, fast scan and all channel scan:
+Two scan methods are supported: fast scan and all channel scan.
 
-* fast scan: in this mode, scan will finish after find match AP even didn't scan all the channel, you can set thresholds for signal and authmode, it will ignore the AP which below the thresholds.
+* fast scan: in this mode, scan finishes right after a matching AP is detected, even if channels are not completely scanned. You can set thresholds for signal strength, as well as select desired authmodes provided by the AP's. The board will ignore AP's that fail to meet mentioned criteria.
```

Comment:
Instead of "board" I propose "Wi-Fi driver" (there are more occurrences of "board" in this text to be replaced)